src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * This class will represent an adding or deleting of a tag for a specific person
+ * that happened during that session.
+ */
+public class Tagging {
+    private static final String ADD_PREFIX = "+ ";
+    private static final String DELETE_PREFIX = "- ";
+
+    private static String personName;
+    private static String tagName;
```

Comment:
Any reason for the `static`? Also, according to the UML, this class should contain Person and Tag objects, not String.